x-pack/plugins/runtime_fields/public/load_editor.tsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import React from 'react';
+import { CoreSetup, OverlayRef } from 'src/core/public';
+
+import { toMountPoint, createKibanaReactContext } from './shared_imports';
+import { LoadEditorResponse, RuntimeField } from './types';
+
+export interface OpenRuntimeFieldEditorProps {
+  onSave(field: RuntimeField): void;
+  defaultValue?: RuntimeField;
+}
+
+export const getRuntimeFieldEditorLoader = (coreSetup: CoreSetup) => async (): Promise<
+  LoadEditorResponse
+> => {
+  const { RuntimeFieldEditorFlyoutContent } = await import('./components');
+  const [core] = await coreSetup.getStartServices();
+  const { uiSettings, overlays, docLinks } = core;
+  const { Provider: KibanaReactContextProvider } = createKibanaReactContext({ uiSettings });
+
+  let overlayRef: OverlayRef | null = null;
+
+  const openEditor = ({ onSave, defaultValue }: OpenRuntimeFieldEditorProps) => {
+    const onSaveField = (field: RuntimeField) => {
+      overlayRef?.close();
+      onSave(field);
+    };
+
+    overlayRef = overlays.openFlyout(
+      toMountPoint(
+        <KibanaReactContextProvider>
+          <RuntimeFieldEditorFlyoutContent
```

Comment:
It is up to consumers to implement the global flyout logic, if they want that, right? I'm thinking it could be cool to mention something like that in the README if we haven't already.

Reply:
So here there is no GlobalFlyout as it makes use of `core.overlays.openFlyout`. If we think it is good to use GlobalFlyout because it adds additional functionality (like drill down to other content) then we can update the README.md with some examples.